src/main/java/org/gestern/gringotts/Gringotts.java

```diff
@@ -9,6 +9,7 @@
 import com.avaje.ebeaninternal.server.ddl.DdlGenerator;
 import com.avaje.ebeaninternal.server.lib.sql.TransactionIsolation;
 import net.milkbowl.vault.economy.Economy;
+import net.tnemc.core.Reserve;
```

Comment:
@creatorfromhell have you try running the plugin without Reserve? I'm really worrying about this.